src/main/scala/cromwell/binding/types/WdlAnyType.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding.types
+
+case object WdlAnyType extends WdlType {
+  val toWdlString: String = s"Any"
+
+  override protected def coercion = {
+    case any: Any =>
```

Comment:
`WdlAnyType` does behave slightly differently than the other `WdlType`s.  For something like `WdlInteger`, the coercion function is "give me one of x different type of objects that are compatible with WdlInteger, and I'll give you a WdlInteger back"

WdlAnyType's coercion semantics are more like "Give me anything at all, and I've got a heuristic that will return you the WdlValue that's able to accept this Any value."  So you give the `coercion()` function a String "foobar", and it returns WdlString("foobar"), you give it a JsNumber(2), it returns WdlInteger(2).

This is used when evaluating literal expressions.  For example, a user might do this in source code:

```
Map[Int, String] first = {"foo": 2, 3.14: "bar"}
Map[Float, String] second = {"foo": "bar"}
```

When we're simply parsing the expression `{"foo": 2, 3.14: "bar"}`, there are two levels of coercion happening.  First it just tries to coerce this into ANY Map[K, V] type.  It uses `Map[Any, Any].coerce` for that.  The first example above would fail this coercion stage.  The second would pass and return a `Map[String, String]`.

The second coercion that happens is the coercion to the target type.  In the example above, `second` starts out coerced as `Map[String, String]`, and then it is `Map[Float, String].coerce` is called on that map.  This step should fail at this stage.